pages/index.js

```diff
@@ -10,10 +11,18 @@ export default function HomePage() {
 
   if (error) return <div>{error.message}</div>;
   if (isLoading) return <div>loading...</div>;
-
+  function getRandomPiece(length) {
+    return Math.floor(Math.random() * length);
+  }
+  const randomPiece = pieces[getRandomPiece(pieces.length)];
+  console.log(randomPiece);
```

Comment:
You should always remove `console.log`'s, to avoid publishing sensitive data by accident.